capstone/client/src/components/ListPage.js

```diff
@@ -329,7 +362,7 @@ class ListPage extends Component {
             </List>
           </Grid>
         </Grid>
-        {this.state.displayZipCodeInput ? <TextField display={this.state.location} id="filled-basic" label="Zip Code" variant="filled" /> : null}
+        {this.state.displayZipCodeInput ? <TextField display={this.state.location} onChange={this.zipCodeChange} id="filled-basic" label="Zip Code" variant="filled" /> : null}
```

Comment:
Add validation to component as user types, something like
https://stackoverflow.com/questions/35901440/how-to-invalidate-a-textfield-in-material-ui